stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1141,6 +1141,7 @@ public class ChatClient internal constructor(
     }
 
     @CheckResult
+    @JvmOverloads
```

Comment:
I think @JvmOverloads is not very helpful for java users as they still can't use the function with arbitrary parameters. E.g. somebody wants to use only last 2 parameters, still needs to pass something for the rest.

Reply:
True but if you want to use only the first parameter, you don't have to pass others. I think that usually, users are going to use the first few parameters and JvmOverloads is helpful in such cases.

Reply:
Ok, I'm just worried that only first 1 or 2 overloads are useful, and others will mostly like not match the exact client's needs.

Reply:
yeah :( I'll leave it for now but, maybe we should somehow group pagination-related parameters into separate classes 🤔